lib/modeler/LintingAnnotations.js

```diff
@@ -0,0 +1,72 @@
+import { groupBy } from 'min-dash';
+
+import { domify } from 'min-dom';
```

Comment:
This is not a dependency right now.

Reply:
👀

Reply:
Added: https://github.com/camunda/linting/pull/11/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R31